lib/matplotlib/ticker.py

```diff
@@ -1991,7 +1991,7 @@ def ge(self, x):
 
 class MaxNLocator(Locator):
     """
-    Select no more than N intervals at nice locations.
+    Select no more than N+1 intervals (i.e. N+2 ticks) at nice locations.
```

Comment:
"Find nice tick locations with no more than N being within the view limits. Locations
beyond the limits are added to support autoscaling."